examples/flutter_gallery/test/accessibility_test.dart

```diff
@@ -255,230 +255,295 @@ void main() {
       final String themeName = themeNames[themeIndex];
 
       testWidgets('backdrop_demo $themeName', (WidgetTester tester) async {
+        final AutomatedTestWidgetsFlutterBinding binding = tester.binding;
+        binding.addTime(const Duration(seconds: 3));
         final SemanticsHandle handle = tester.ensureSemantics();
         await tester.pumpWidget(MaterialApp(theme: theme, home: BackdropDemo()));
         await expectLater(tester, meetsGuideline(textContrastGuideline));
         handle.dispose();
       });
 
       testWidgets('bottom_app_bar_demo $themeName', (WidgetTester tester) async {
+        final AutomatedTestWidgetsFlutterBinding binding = tester.binding;
+        binding.addTime(const Duration(seconds: 3));
         final SemanticsHandle handle = tester.ensureSemantics();
         await tester.pumpWidget(MaterialApp(theme: theme, home: BottomAppBarDemo()));
         await expectLater(tester, meetsGuideline(textContrastGuideline));
         handle.dispose();
       }, skip: true); // https://github.com/flutter/flutter/issues/21651
 
       testWidgets('bottom_navigation_demo $themeName', (WidgetTester tester) async {
+        final AutomatedTestWidgetsFlutterBinding binding = tester.binding;
+        binding.addTime(const Duration(seconds: 3));
         final SemanticsHandle handle = tester.ensureSemantics();
         await tester.pumpWidget(MaterialApp(theme: theme, home: BottomNavigationDemo()));
         await expectLater(tester, meetsGuideline(textContrastGuideline));
         handle.dispose();
       });
 
       testWidgets('buttons_demo $themeName', (WidgetTester tester) async {
+        final AutomatedTestWidgetsFlutterBinding binding = tester.binding;
+        binding.addTime(const Duration(seconds: 3));
         final SemanticsHandle handle = tester.ensureSemantics();
         await tester.pumpWidget(MaterialApp(theme: theme, home: ButtonsDemo()));
         await expectLater(tester, meetsGuideline(textContrastGuideline));
         handle.dispose();
       }, skip: true); // https://github.com/flutter/flutter/issues/21647
 
       testWidgets('cards_demo $themeName', (WidgetTester tester) async {
+        final AutomatedTestWidgetsFlutterBinding binding = tester.binding;
+        binding.addTime(const Duration(seconds: 3));
         final SemanticsHandle handle = tester.ensureSemantics();
         await tester.pumpWidget(MaterialApp(theme: theme, home: CardsDemo()));
         await expectLater(tester, meetsGuideline(textContrastGuideline));
         handle.dispose();
       }, skip: true); // https://github.com/flutter/flutter/issues/21651
 
       testWidgets('chip_demo $themeName', (WidgetTester tester) async {
+                final AutomatedTestWidgetsFlutterBinding binding = tester.binding;         binding.addTime(const Duration(seconds: 3));
```

Comment:
Formatting looks off here - indented too far and two lines of code in one line.

(otherwise, LGTM)

Reply:
Woops, fixed!